source/draw/Canvas.ooc

```diff
@@ -38,28 +36,33 @@ Canvas: abstract class {
 		this opacity = 1.0f
 		this interpolationMode = InterpolationMode Fast
 	}
-	drawPoint: virtual func (position: FloatPoint2D) {
+	drawPoint: virtual func ~white (position: FloatPoint2D) { this drawPoint(position, Pen new(ColorRgba white)) }
```

Comment:
Why not `drawPoint: virtual func (position: FloatPoint2D, pen := Pen new(ColorRgba white))` ?